cspell.json

```diff
@@ -68,376 +63,333 @@
-        "ericsakmar",
-        "Ericsburgh",
+        "eneman",
+        "enoent",
+        "ericsburgh",
         "errored",
-        "Esben",
+        "esben",
         "esbenp",
-        "eslint's",
         "eslump",
         "esproposal",
         "estree",
         "esutils",
         "eval",
-        "evilebottnawi",
         "execa",
         "fbglyph",
-        "Ficarra",
+        "fbid",
+        "ficarra",
         "filepath",
-        "Filipe",
+        "filipe",
         "finalizer",
-        "Fiorini",
-        "flxwu",
+        "fiorini",
         "fnames",
         "formatprg",
-        "Friedly",
+        "friedly",
+        "frobble",
         "fuzzer",
-        "gavinjoyce",
-        "Georgii",
+        "georgii",
         "gettin",
         "git's",
         "gitattributes",
         "githook",
-        "gitignore",
         "gitkeep",
-        "Gitter",
+        "gitter",
         "glimmerjs",
         "globbing",
         "globby",
         "gofmt",
-        "googlegroups",
-        "googlemaps",
-        "Gregor",
-        "hackily",
-        "haggholm",
-        "Hampus",
+        "gregor",
+        "gtag",
+        "hampus",
         "hardcoded",
         "hardline",
         "hardlines",
-        "harel",
         "hashbang",
-        "Hawken",
-        "Hengles",
-        "Hersevoort",
+        "hawken",
+        "hengles",
+        "hersevoort",
+        "hljs",
         "hlsson",
-        "hongrich",
-        "Horky",
+        "horky",
         "hotpink",
         "hsla",
         "htmlblock",
         "htmlhint",
         "hugomrdias",
         "hwba",
         "iarna",
-        "ICSS",
+        "icss",
         "idempotence",
-        "IIFE",
-        "IIFEs",
+        "iife",
+        "iifes",
         "ikatyang",
-        "Ilya",
+        "ilya",
         "impltype",
         "importee",
         "importmap",
         "indentable",
-        "indexof",
         "infc",
         "instanceof",
-        "Intelli",
+        "intelli",
         "intrinsics",
         "jackyho",
-        "Jakefile",
+        "jakefile",
         "jakegavin",
-        "jbrown",
         "jetbrains",
         "jlongster",
-        "jnwng",
-        "Joar",
+        "joar",
         "josephfrazier",
-        "Joun",
-        "jounqin",
-        "jridgewell",
-        "jscodefmt",
         "jsesc",
         "jsfmt",
-        "JSXs",
-        "junit",
-        "jwbay",
-        "kachkaev",
+        "jsxs",
         "kalmanb",
-        "Karimov",
-        "Kassens",
-        "Kasturi",
-        "Kearney",
-        "keyof",
-        "Khatri",
-        "koba",
+        "karimov",
+        "kassens",
+        "kasturi",
+        "kearney",
+        "khatri",
         "lcov",
         "libdef",
         "linearize",
         "linebreak",
         "linebreaks",
         "literalline",
         "literallines",
-        "lockfile",
         "loglevel",
         "lowercased",
         "lowercasing",
         "lydell",
-        "macos",
```

Comment:
Seems like `.github/` is still excluded. "macos" is from the GitHub Actions files.

Reply:
Seems hidden files excluded by default